man/subscription-manager.8

```diff
@@ -60,14 +60,14 @@ has specific options available for each command, depending on what operation is
 
 .PP
 .B Note:
-Please note that using commands that require providing a username using 
+Please note that using commands that require providing a username using
 .B --username
-, a password using 
-.B --password 
-, an organization using 
-.B --org 
-, or environments using 
-.B --environments 
+, a password using
+.B --password
+, an organization using
+.B --org
+, or environments using
+.B --environments
```

Comment:
note those whitespace changes are unrelated to the change; if would it be possible to avoid them, please do